fluster/codec.py

```diff
@@ -40,3 +40,8 @@ class OutputFormat(Enum):
     NONE = "None"
     YUV420P = "yuv420p"
     YUV420P10LE = "yuv420p10le"
+    YUV420P12LE = "yuv420p12le"
+    YUV422P10LE = "yuv422p10le"
+    YUV422P12LE = "yuv422p12le"
+    YUV444P10LE = "yuv444p10le"
+    YUV444P12LE = "yuv444p12le"
```

Comment:
Best would be to rebase on top of https://github.com/fluendo/fluster/pull/117 ?

Reply:
I just noticed that apparently videocodectestsink does not have support for some of these?

running fluster with the libvpx is 6/6, but gstreamer-libvpx is only 2/6 because of incompatible caps

Reply:
I added the 10bit one yesterday, we need to add the 444/12bit now.

Reply:
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/3356